src/main/java/org/opensrp/config/RedisConfig.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * 
+ */
+package org.opensrp.config;
+
+import org.apache.commons.pool2.impl.GenericObjectPoolConfig;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.redis.connection.RedisConnectionFactory;
+import org.springframework.data.redis.connection.RedisStandaloneConfiguration;
+import org.springframework.data.redis.connection.jedis.JedisClientConfiguration;
+import org.springframework.data.redis.connection.jedis.JedisConnectionFactory;
+import org.springframework.data.redis.core.RedisTemplate;
+
+/**
+ * @author Samuel Githengi created on 05/11/20
+ */
+@Configuration
+public class RedisConfig {
```

Comment:
This class should be moved in whole from web or maintained in web

Reply:
Moved file to server-core

Reply:
Done